aiohttp/worker.py

```diff
@@ -34,7 +34,9 @@ def run(self):
         self._runner = ensure_future(self._run(), loop=self.loop)
 
         try:
-            self.loop.run_until_complete(self._runner)
+            self.loop.run_until_complete(asyncio.gather(self._runner,
```

Comment:
I suspect parallel execution by `gather` is not desired.
At least it works different that `web.run_app()`.
Better to wait for `self.wsgi.startup()` and than execute `self._runner` for opening server-side socket etc.